data/practiceFile

```diff
@@ -0,0 +1 @@
+{"Vehicle":{"Car":"Blue Tacoma"}}
```

Comment:
I think we may want to .gitignore this directory before we have large config files here. Up to you.

Reply:
Actually, this decision is mostly up to you. I thought we already discussed that the general format of the configs wan't a breach in privacy or important internal information. Because we don't know exactly how all these fields are actually named, I don't think that it would be necessary to put the folder in the .gitignore. The way the fields in the mock configs are named won't be the same way they are named internally.

Reply:
I am okay with this for now. I am not concerned with the security here, just that, in the future, PRs will be harder to parse through if they have a lot of test data in them.